services/github/github-directory-file-count.tester.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
```

Comment:
We can use the `isMetric` helper here for validating the badge messages
```suggestion

const { isMetric } = require('../test-validators')
```